experiments/ssl4eo/landsat/plot_landsat_bands.py

```diff
@@ -46,7 +46,9 @@
 df = df.iloc[::-1]
 
 fig, ax = plt.subplots(figsize=(5.5, args.fig_height))
-ax1, ax2 = fig.subplots(nrows=1, ncols=2, gridspec_kw={"width_ratios": [3, 1]})
+ax1, ax2 = fig.subplots(
+    nrows=1, ncols=2, gridspec_kw={"width_ratios": [3, 1]}
+)  # type: ignore[misc]
```

Comment:
All type ignores are things that are not yet type hinted. If I remember I'll submit a PR someday, but right now my dev environment is broken.